src/main/java/frc/robot/Main.java

```diff
@@ -13,10 +13,16 @@
 
 import edu.wpi.first.wpilibj.RobotBase;
 
-/* Probably don't put code here, most likely put it in RobotContainer. */
+// Hey! You probably don't want to put code here! Your might looking for RobotContainer.
```

Comment:
Per team 937 conventions, `//` comments are only for commenting out code, not leaving notes to other programmers.